<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordRank</class>
 <widget class="QDialog" name="RecordRank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="check">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>98</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="firstLayout">
    <item>
     <widget class="QLabel" name="firstName">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>姓：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="firstNameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="lastLayout">
    <item>
     <widget class="QLabel" name="lastName">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="lastNameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="wpmLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>461</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>速度：       字 / 秒</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check</sender>
   <signal>accepted()</signal>
   <receiver>RecordRank</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check</sender>
   <signal>rejected()</signal>
   <receiver>RecordRank</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
